### PR TITLE
Put validation under commit.canExecute

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: src/plugins/validation/validator.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import PluginComponent from '../plugin.component';
-import * as validationService from './validation.service';
+import * as validationService from '@grid/core/validation/validation.service';
```
Comment (vkorolev) [inline src/plugins/validation/validator.js:2]: is it more readable?:)

File: src/core/edit/edit.cell.view.js [hunk 2]
```diff
@@ -105,15 +106,19 @@ export class EditCellView extends View {
 				priority: 1,
 				source: 'edit.cell.view',
 				shortcut: this.shortcutFactory('commit'),
-				// TODO: add validation support
 				canExecute: cell => {
 					cell = cell || this.editor.cell;
+					const context = this.contextFactory(cell);
+					const key = context.column.key;
+					const validator = validationService.createValidator(model.validation().rules, key);
+					const valid = validator.validate({[key]: this.value});
```
Comment (klumba12) [inline src/core/edit/edit.cell.view.js]: this should be the in the last && so there is a chance to not call it

File: src/plugins/validation/validator.js (hunk 1 above)
Comment (etokareva) [reply src/plugins/validation/validator.js:2]: No)) But I moved validation.service to core. So it is more readable than '../../core/validation/validation.service'

Comment (vkorolev) [reply src/plugins/validation/validator.js:2]: Oh, i see, sorry

**Review by vkorolev — APPROVED** [review]

**Pull request merged**